src/button-dropdown/__integ__/button-dropdown-events.test.ts

```diff
@@ -76,4 +78,21 @@ describe('clicking on a ButtonDropdown item', () => {
       await expect(page.getLocation()).resolves.toEqual(oldLocation);
     })
   );
+  test(
```

Comment:
Since this feature is meant for an internal usage of property filter only, I would be more inclined to test this as part of property filter functionality, rather than making a case for a standalone button dropdown with only main action which in reality we don't expose as a user experience —it is an implementation detail. But this is not a blocker